### ISSUE TITLE
Unable to Download Epic Games Installer

### ISSUE THREAD

**Issue by trentnbauer** [lancachenet/sniproxy#11]
**ISSUE:**
I am attempting to download the Epic Games MSI from here: https://www.epicgames.com/store/en-US/download , it shows a ERR_TIMED_OUT.

**TROUBLESHOOTING:**
I can download the MSI perfectly fine from a network that does not have lancache configured
Steam is caching as expected
I have removed and reconfigured SNIProxy

Command I used to run SNIProxy
`docker run --restart unless-stopped --name sniproxy --detach -p 10.0.70.30:443:443 -e UPSTREAM_DNS="1.1.1.1 1.0.0.1" lancachenet/sniproxy:latest`

NSlookup for launcher-public-service-prod06.ol.epicgames.com
```
from Home (lancache)
Non-authoritative answer:
Name:    launcher-svc-prod06-pub-1212613127.us-east-1.elb.amazonaws.com
Addresses:  34.204.154.85
          54.88.124.231
          54.172.26.197
          34.202.39.50
Aliases:  launcher-public-service-prod06.ol.epicgames.com
```

Different Network (no lancache)
```
Non-authoritative answer:
Name:    launcher-svc-prod06-pub-1212613127.us-east-1.elb.amazonaws.com
Addresses:  54.172.26.197
          54.88.124.231
          34.202.39.50
          34.204.154.85
Aliases:  launcher-public-service-prod06.ol.epicgames.com
```

**ENVIRONMENT**
Docker is running on an Ubuntu 18 virtual machine
```
administrator@docker:~$ docker container ls
CONTAINER ID        IMAGE                             COMMAND                  CREATED             STATUS              PORTS                            NAMES
7423420c7db6        lancachenet/sniproxy:latest       "/scripts/bootstrap.…"   25 minutes ago      Up 25 minutes       10.0.70.30:443->443/tcp          sniproxy
481f2bf9fccb        lancachenet/lancache-dns:latest   "/bin/bash -e /init/…"   3 days ago          Up 2 days           10.0.70.30:53->53/udp            lancache-dns
fea967eb69f8        lancachenet/monolithic:latest     "/bin/bash -e /init/…"   3 days ago          Up 2 days           10.0.70.30:80->80/tcp, 443/tcp   lancache
administrator@docker:~$ 
```

**Comment by unspec** [comment]
Hi,

Are you still having this issue? 

I'm able to download this file fine from behind a cache setup, however *.ol.epicgames.com is not included in https://github.com/uklans/cache-domains/blob/master/epicgames.txt and so isn't being pointed to the IP of your cache / sni proxy (as your nslookup shows) - so it shouldn't be interfering with a simple file download.

**Comment by unspec** [comment]
Closing this issue due to inactivity, feel free to reopen if needed

**Comment by mat32323** [comment]
Hi, 

Just thought I'd leave a comment here because this is the only thread I've found on this issue. I had exactly this problem but have managed to resolve it

The issue was that in my lancache settings I had set my routers IP as the DNS forwarding server and on the router set the lancache server as the primary DNS

This was actually working fine for the most part, caching steam games worked okay and general web browsing was fine. The only issue I had was with downloading the Epic Games launcher and it then not caching games. I would assume only certain routers have this problem. When using DHCP my router will give a DNS server of itself and no secondary DNS. From there it would send the DNS lookup out to one of the two specified DNS servers on the router

The fix if you have this issue is to change your lancache DNS settings to forward to an address other than your router, such as Google's DNS (8.8.8.8)
